### PR TITLE
[59] Implement dev.cfp_v3 lambdas and DB

### LINKED ISSUE #59: Implement and deploy dev version of codeforpoznan.pl_v3 project
Standard setup - two lambdas, one for api and second for migrations. Both hidden behind API Gateway. CloudFront in front of API Gateway for managing two origins - S3 and Lambda.

### PR DESCRIPTION
**Pull request by arturtamborski**
This was _not_ applied yet, I only provide it as a starting point.

### CODE REVIEW

**Comment by arturtamborski** [comment]
I think its ready to be applied, waiting for review.

File: dev_codeforpoznan_pl_v3.tf [hunk 1]
```diff
@@ -0,0 +1,159 @@
+resource "aws_iam_user" "dev_codeforpoznan_pl_v3" {
```
Comment (magul) [inline dev_codeforpoznan_pl_v3.tf]: use `./user` module

File: dev_codeforpoznan_pl_v3.tf [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+resource "random_password" "dev_codeforpoznan_pl_v3_secret_key" {
+  length  = 50
+  special = false
+}
+
+module dev_codeforpoznan_pl_v3_migration {
+  source = "./lambda"
+
+  name            = "dev_codeforpoznan_pl_v3"
```
Comment (magul) [inline dev_codeforpoznan_pl_v3.tf]: that should end with `_migration` (to have s3 object separate for migration and for API)

File: dev_codeforpoznan_pl_v3.tf [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+resource "aws_iam_policy" "dev_codeforpoznan_pl_v3_ses_policy" {
+  name       = "dev_codeforpoznan_pl_v3_ses_policy"
+  policy     = data.aws_iam_policy_document.dev_codeforpoznan_pl_v3_ses_policy.json
+  depends_on = [module.dev_codeforpoznan_pl_v3_mailing_identity.domain_identity]
+}
+
+module dev_codeforpoznan_pl_v3_serverless_api {
+  source = "./serverless_api"
+
+  name                = "dev_codeforpoznan_pl_v3"
```
Comment (magul) [inline dev_codeforpoznan_pl_v3.tf:95]: ditto

**Review by magul — CHANGES_REQUESTED** [review]
Minor issues, rest looks fine.

**Comment by arturtamborski** [comment]
updated!

**Review by magul — DISMISSED** [review]
this file should be merged with https://github.com/CodeForPoznan/Infrastructure/blob/d655c0a323f76e9c523bd81cf3515f703d76f0d9/codeforpoznan_pl_v3.tf

**Comment by arturtamborski** [comment]
Update: this PR is now final and ready to be reviewed again! I also applied the change listed here, so all resources listed in diff are either moved over from `codeforpoznan_v3.tf` or created.

Please mind that there is a breaking change (provider specifier in ssl module) which can't work without https://github.com/CodeForPoznan/Infrastructure/pull/72), so that linked PR should be merged first and then this can go cleanly.

**Review by Maciek246 — APPROVED** [review]

**Pull request merged**